app/controllers/lessons_controller.rb

```diff
@@ -14,7 +14,8 @@ def new
 
   def create
     @lesson = Lesson.new(lesson_params)
-    @lesson.studio = current_user.studio
+    @lesson.user = current_user
+    @lesson.studio_id = current_user.studio.id
```

Comment:
here im a bit unsure.. i think we are creating a lesson that does not belong to the current user but to a teacher, so im not sure if we need to assign it to the current user.. lets discuss!

Reply:
It was the only way with Guido yesterday to get it to work as a lesson belongs to a user